lr_face/experiments.py

```diff
@@ -193,3 +194,30 @@ def data_keys(self) -> List[str]:
         :return: List[str]
         """
         return list(set(k for v in DATA['all'].values() for k in v.keys()))
+
+
+def dataset_pairs(dataset):
```

Comment:
so we now have dataset_pairs, dataset.pairs, get_pairs and make_pairs
could we have clearer naming?